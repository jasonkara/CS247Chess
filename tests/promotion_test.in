game human human
move g2 g4
move h7 h5
move g4 h5
move g7 g5
move g1 f3
move g5 g4
move a2 a3
move g4 g3
move a3 a4
move g3 g2
move a4 a5
move g2 g1 p 
q