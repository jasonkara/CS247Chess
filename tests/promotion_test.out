8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _P_
3 _ _ _ _ 
2 PPPPPP P
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr
7 ppppppp 
6  _ _ _ _
5 _ _ _ _p
4  _ _ _P_
3 _ _ _ _ 
2 PPPPPP P
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr
7 ppppppp 
6  _ _ _ _
5 _ _ _ _P
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPP P
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr
7 pppppp_ 
6  _ _ _ _
5 _ _ _ pP
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPP P
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr
7 pppppp_ 
6  _ _ _ _
5 _ _ _ pP
4  _ _ _ _
3 _ _ _N_ 
2 PPPPPP P
1 RNBQKB_R

  abcdefgh
8 rnbqkbnr
7 pppppp_ 
6  _ _ _ _
5 _ _ _ _P
4  _ _ _p_
3 _ _ _N_ 
2 PPPPPP P
1 RNBQKB_R

  abcdefgh
8 rnbqkbnr
7 pppppp_ 
6  _ _ _ _
5 _ _ _ _P
4  _ _ _p_
3 P _ _N_ 
2  PPPPP P
1 RNBQKB_R

  abcdefgh
8 rnbqkbnr
7 pppppp_ 
6  _ _ _ _
5 _ _ _ _P
4  _ _ _ _
3 P _ _Np 
2  PPPPP P
1 RNBQKB_R

  abcdefgh
8 rnbqkbnr
7 pppppp_ 
6  _ _ _ _
5 _ _ _ _P
4 P_ _ _ _
3 _ _ _Np 
2  PPPPP P
1 RNBQKB_R

  abcdefgh
8 rnbqkbnr
7 pppppp_ 
6  _ _ _ _
5 _ _ _ _P
4 P_ _ _ _
3 _ _ _N_ 
2  PPPPPpP
1 RNBQKB_R

  abcdefgh
8 rnbqkbnr
7 pppppp_ 
6  _ _ _ _
5 P _ _ _P
4  _ _ _ _
3 _ _ _N_ 
2  PPPPPpP
1 RNBQKB_R

  abcdefgh
8 rnbqkbnr
7 pppppp_ 
6  _ _ _ _
5 P _ _ _P
4  _ _ _ _
3 _ _ _N_ 
2  PPPPP P
1 RNBQKBqR

  abcdefgh
Final Score:
White: 0
Black: 0
